packages/blocks/stopwatch/src/app.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState, useCallback } from "react";
+import { BlockComponent } from "blockprotocol/react";
+
+type AppProps = {
+  start: Date;
+  laps: number[];
+};
+
+function formatDuration(duration: number) {
+  const MILLISECONDS = 1;
+  const SECONDS = 1000 * MILLISECONDS;
+  const MINUTES = 60 * SECONDS;
+  const HOURS = 60 * MINUTES;
+  const DAYS = 24 * HOURS;
```

Comment:
Also optional: because these are constants, we can place them before the function. We also don't need to use `UPPER_CASE` but it's a matter of taste 🙂

Reply:
I prefer `UPPER_CASE` for constants. I put them into the function because they are not used anywhere else. Happy to change for the sake of consistency 🙂

Reply:
We were discussing top-level `UPPER_CASE` constants, not the ones inside closures. That's another source of entropy in the codebase, which is small on its own, but they add up.

Reply:
Yes, I was talking about this:

> We also don't need to use UPPER_CASE but it's a matter of tast

Let's stop raising it in individual PRs please. It's wasting time when we've said we'll leave it up to personal taste – more time than any theoretical confusion about what they mean, which hasn't arisen.

Reply:
Oh, are you saying that the thread related to `UPPER_CASE` in one place, but not another? It doesn't say that anywhere – we can leave it up to personal taste everywhere.